src/main/java/core/basesyntax/AllColors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum AllColors {
+    Blue,
+    Red,
+    Green,
+    White,
+    Black,
+    Brown,
+    Orange,
+    Purple,
+    Yellow
+
```

Comment:
Don’t forget how to name enums according to style guide (the same with naming of constants)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names